lib/utils/device/live.ts

```diff
@@ -428,6 +406,14 @@ export class LivepushManager {
 				return [];
 		}
 	}
+
+	// For each service, get a debounced function
+	private getDebouncedEventHandle = _.memoize((serviceName: string) => {
```

Comment:
```suggestion
	private getDebouncedEventHandleFunc = _.memoize((serviceName: string) => {
```
Slight name change to make it obvious it's a function (I looked at where it was called first, just a but more explicit).